registry/remote/repository.go

```diff
@@ -214,6 +219,14 @@ func (r *Repository) Push(ctx context.Context, expected ocispec.Descriptor, cont
 	return r.blobStore(expected).Push(ctx, expected, content)
 }
 
+// Mount makes the blob with the given digest in fromRepo
+// available in the repository signified by the receiver.
+//
+// This avoids the need to pull content down from fromRepo only to push it to r.
+func (r *Repository) Mount(ctx context.Context, desc ocispec.Descriptor, fromRepo string) error {
+	return r.Blobs().(*blobStore).Mount(ctx, desc, fromRepo)
```

Comment:
I'll change it, but FWIW I actually started by doing `registry.Mounter` but thought it was actually better to use `*blobStore` because we know statically in this package that it's got to be `*blobStore` and we only care about being able to call Mount (and the code is arguably more obvious this way, as there are less levels of indirection in play).

Reply:
done